app/workers/set_asset_size_worker.rb

```diff
@@ -0,0 +1,8 @@
+class SetAssetSizeWorker
+  include Sidekiq::Worker
+  sidekiq_options queue: 'low_priority'
+
+  def perform(asset_id)
+    Asset.unscoped.find(asset_id).set_size_from_etag
```

Comment:
I'd be inclined to inline the `Asset#set_size_from_etag` method into the worker, because I don't think it's used by any production code and so it should minimise the chance that the code gets left behind when we remove this one-off migration.

Reply:
Are you confident that the call to `Asset#save` doesn't have any side-effects? As I mentioned in Slack earlier, I'd be inclined to use `Mongoid::Persistable::Settable#set` here c.f. https://github.com/alphagov/asset-manager/blob/314ddfcdee40e2e25273c0acb3327f3fd9771a99/app/workers/asset_file_metadata_worker.rb#L6-L10.